### PR TITLE
update security.md

### PR DESCRIPTION
**Pull request by Soapy7261**
i should probably also change the email but idk what the correct email is :c

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Amongst other things, things that counts as a security vulnerability include, bu
 ### How to report it
 We ask you to report any and all vulnerabilities to us in one of the following ways:
 - Sending a mail to [joshuaslui0203@gmail.com](mailto:joshuaslui0203@gmail.com)
```
Comment (JoshuaSlui) [inline SECURITY.md]:
```suggestion
- Sending a mail to [support@wolfsvale.tech](mailto:support@wolfsvale.tech)
```

**Review by JoshuaSlui — CHANGES_REQUESTED** [review]

**Comment by Soapy7261** [comment]
k

**Comment by Soapy7261** [comment]
please :pray:

**Comment by JoshuaSlui** [comment]
CodeQL just doesn't know what to do anymore

**Pull request merged**